context/app/static/js/components/workspaces/statusCodes.js

```diff
@@ -0,0 +1,49 @@
+// Workspaces:
+
+const workspaceStatus = {
+  active: {
+    isDone: false,
+  },
+  idle: {
+    isDone: false,
+  },
+  deleting: {
+    isDone: true,
+  },
+  error: {
+    isDone: true,
+  },
+};
+
+function validateWorkspaceStatus(status) {
+  const expected = Object.keys(workspaceStatus);
+  if (!expected.includes(status)) {
```

Comment:
```suggestion
  if (!(status in workspaceStatus)) {
```

Reply:
will do locally...

Reply:
... actually, I still want `expected` for the error message, so I'll keep this as is. Otherwise, I agreed this would be simpler.